src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -2748,8 +2748,8 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         {
             _bellDarkAnimation = Window::Current().Compositor().CreateScalarKeyFrameAnimation();
             // reversing the order of the intensity values produces a similar effect as the light version
-            _bellDarkAnimation.InsertKeyFrame(0.0, 1.0);
-            _bellDarkAnimation.InsertKeyFrame(1.0, 2.0);
+            _bellDarkAnimation.InsertKeyFrame(0.0f, 1.0f);
+            _bellDarkAnimation.InsertKeyFrame(1.0f, 2.0f);

```

Comment:
Curiously with `/fp:fast` this code compiles without warnings. Even in debug builds!
See: https://godbolt.org/z/exdTnf9sT

Reply:
Is it _supposed_ to fire warnings? It isn't a meaningful truncation -- it's a knowable compile time constant. I might say that the diagnostic is meaningless there.

After all, there's an automatic warning-free conversion when you call `foo(short)` with `17`. Why shouldn't there be one here? Even though the typeless number value _could_ _conceivably_ be larger than a short.

Reply:
I'm honestly not sure and I don't get it either. But since there's only a single spot where this warning exists, I feel like keeping the warning enabled is the safer choice.